### ISSUE TITLE
Support GRPC health checks

### ISSUE THREAD

**Issue by jeckhart** [nestjs/terminus#183]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ ] Bug report
[X] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

Related to nestjs/nest#2214 , we would like to use the [GRPC Health Check](https://github.com/grpc/grpc/blob/master/doc/health-checking.md) protocol in our microservices. I'm not sure if there is a easy way to hook [nestjs/terminus](https://github.com/nestjs/terminus) into the GRPC transport in Nest, but it would be a big win for us.

## What is the motivation / use case for changing the behavior?

Rather than reinvent the wheel and/or include our own separate health check handler, we would like to use the health check that is part of the GRPC spec. Several languages include built-in support and there are tools already developed to use the health check in the spec.

## Environment

<pre><code>
Nest version: all

 
For Tooling issues:
- Node version: all
- Platform:  all

Others:

</code></pre>

**Comment by BrunnerLivio** [comment]
There is a way to check if your microservice connects to your client using gRPC, like this
```typescript
async () =>
          microservice.pingCheck('grpc', {
            transport: Transport.GRPC,
            options: {
              ...
            },
          }),
```

See more: https://github.com/nestjs/terminus/blob/master/sample/002-microservice-app/src/health.module.ts

It does not check the `grpc.health.v1.` namespace explicitly though. So I think a `microservice.checkGRPC` which check this namespace would be a good idea!

**Comment by BrunnerLivio** [comment]
Added with #191 and soon to be released with 6.2.0
